libs/core/resource_partitioner/tests/unit/scheduler_binding_check.cpp

```diff
@@ -68,7 +68,9 @@ void threadLoop()
         auto exec = hpx::execution::parallel_executor(
             hpx::threads::thread_priority::bound,
             hpx::threads::thread_stacksize::default_,
-            hpx::threads::thread_schedule_hint(std::int16_t(i % threads)));
+            hpx::threads::thread_schedule_hint(
+                hpx::threads::thread_schedule_hint_mode::thread,
+                std::int16_t(i % threads), true));
```

Comment:
Is the runs-as-child hint true for testing or is it meant to stay?

Reply:
I have disabled direct execution for the shared_priority_scheduler, this is a test that things still work even if a thread has a corresponding hint.

Reply:
The hint should essentially just be ignored by the shared priority scheduler, correct?

Reply:
yes, indeed.